sycl/include/sycl/ext/intel/esimd/memory.hpp

```diff
@@ -5836,106 +5853,6 @@ atomic_update(AccessorTy acc, simd<Toffset, N> byte_offset, simd<T, N> src0,
   return atomic_update<Op, T, N>(acc, byte_offset, src0, mask, props);
 }
 
-/// simd<T, N>
```

Comment:
if the function is removed here, then the comment at https://github.com/intel/llvm/blob/339e376f48684e5321b9be5a6665dfb129815606/sycl/include/sycl/ext/intel/esimd/memory.hpp#L5693 should also be removed.

Please also compact the numbers after removal: e.g. 1,2,3,4,5,6, after removal of 3 and 4 become: 1,2,5,6,..., please rename 5,6,... to 3,4,...